Sources/MuxUploadSDK/Upload/ChunkedFileUploader.swift

```diff
@@ -88,15 +89,33 @@ class ChunkedFileUploader {
             do {
                 // It's fine if it's already open, that's handled by ignoring the call
                 try file.openFile(fileURL: uploadInfo.videoFile)
+                let fileSize = file.fileSize
                 try file.seekTo(byte: lastByte)
                 let result = try await makeWorker().performUpload()
                 file.close()
-                
+
                 let success = UploadResult(
                     finalProgress: result.progress,
                     startTime: result.startTime,
                     finishTime: result.updateTime
                 )
+
+                let asset = AVAsset(url: uploadInfo.videoFile)
+
+                var duration: CMTime
+                if #available(iOS 16, *) {
```

Comment:
This [doc](https://developer.apple.com/documentation/avfoundation/avasynchronouskeyvalueloading/3747326-load) makes me believe this can be `if #available(iOS 15, *) {`.